.github/workflows/e2e-test-no-domain.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
       AWS_DEFAULT_REGION: us-east-1
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_ADDITIONAL_ROLE: ${{ secrets.AWS_ADDITIONAL_ROLE }}
```

Comment:
This is not as part of this PR but I think is good to add `AWS_ADDITIONAL_ROLE` to Helm chart e2e-test as well after we merge this and made a terraform release.

Reply:
Created [CLIP-1629](https://bulldog.internal.atlassian.com/browse/CLIP-1629) to track this.